### PR TITLE
Update akka persistence version to 2.4.7

### PR DESCRIPTION
**Pull request by zapletal-martin**

### CODE REVIEW

File: src/main/scala/akka/persistence/journal/JournalSpec.scala [hunk 1]
```diff
@@ -38,97 +39,103 @@ trait JournalSpec extends PluginSpec {
...
       subscribe[DeleteMessagesTo](sub.ref)
       journal ! cmd
       sub.expectMsg(cmd)
 
+      receiverProbe.expectMsg(DeleteMessagesSuccess(3))
       journal ! ReplayMessages(1, Long.MaxValue, Long.MaxValue, pid, receiverProbe.ref)
       List(4, 5) foreach { i => receiverProbe.expectMsg(replayedMessage(i)) }
     }
+
+    // TODO: The below is not longer supported
```
Comment (zapletal-martin) [inline src/main/scala/akka/persistence/journal/JournalSpec.scala:119]: AFAIK logical deletes are no longer supported and ReadHighestSequenceNr is no longer in the public api.

**Comment by zapletal-martin** [comment]
Likely not need and superseded by https://github.com/akka/akka/tree/master/akka-persistence-tck

**Comment by krasserm** [comment]
Thanks for your contribution @zapletal-martin. As you already mentioned, this test kit is superseded by akka-persistence-tck. Do you nevertheless need these changes in any of your dependencies? How do you want to proceed?

**Comment by krasserm** [comment]
oh, just realized that you already closed this ticket 😃

**Pull request closed without merging**